pkg/gardenlet/operation/botanist/vpnshoot.go

```diff
@@ -37,7 +38,13 @@ func (b *Botanist) DefaultVPNShoot() (component.DeployWaiter, error) {
 		HighAvailabilityNumberOfSeedServers:  b.Shoot.VPNHighAvailabilityNumberOfSeedServers,
 		HighAvailabilityNumberOfShootClients: b.Shoot.VPNHighAvailabilityNumberOfShootClients,
 		DisableNewVPN:                        !b.Shoot.UsesNewVPN,
-		SeedPodNetwork:                       b.Seed.GetInfo().Spec.Networks.Pods,
+		SeedPodNetworkV4:                     b.Seed.GetInfo().Spec.Networks.Pods,
+	}
+
+	if !gardencorev1beta1.IsIPv6SingleStack(b.Shoot.GetInfo().Spec.Networking.IPFamilies) {
+		values.ShootPodNetworkV4 = b.Shoot.Networks.PodsV4String()
```

Comment:
@ScheererJ I've tried applying your suggestion of using `b.Shoot.Networks` instead of `b.Shoot.GetInfo().Spec.Networking`. I'm getting a nil pointer panic here. The issue seems to be that work non-workerless shoots the `Networks` field is actually populated at a later point in time during [reconcile](https://github.com/gardener/gardener/blob/master/pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go#L188) and waiting for the infrastructure, but the vpnshoot object is already created when [the botanist is created](https://github.com/gardener/gardener/blob/master/pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go#L73).